hotels/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+
+class Category(models.Model):
+    name         = models.CharField(max_length = 45)
+    location     = models.ManyToManyField('Location', through = 'CategoryLocation')
+
+    class Meta:
+        db_table = 'categories'
+
+class Location(models.Model):
+    name         = models.CharField(max_length = 45)
+
+    class Meta:
+        db_table = 'locations'
+
+class CategoryLocation(models.Model):
+    category     = models.ForeignKey('Category', on_delete = models.CASCADE)
+    location     = models.ForeignKey('Location', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = 'category_locations'
+
+class Hotel(models.Model):
+    name            = models.CharField(max_length = 45)
+    address         = models.CharField(max_length = 500)
+    thumbnail_image = models.URLField(max_length = 2000)
+    longitude       = models.DecimalField(max_digits = 26, decimal_places = 20)
+    latitude        = models.DecimalField(max_digits = 26, decimal_places = 20)
+    category        = models.ForeignKey('Category', on_delete = models.CASCADE)
+    location        = models.ForeignKey('Location', on_delete = models.SET_NULL, null = True)
+    hotel_star      = models.ForeignKey('HotelStar', on_delete = models.SET_NULL, null = True)
+
+    class Meta:
+        db_table    = 'hotels'
+
+class HotelStar(models.Model):
+    star         = models.IntegerField()
+
+    class Meta:
+        db_table = 'hotel_stars'
+
+class HotelImage(models.Model):
+    hotel_image  = models.URLField(max_length = 2000)
+    hotel        = models.ForeignKey('Hotel', on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = 'hotel_images'
+
+class RoomType(models.Model):
+    name            = models.CharField(max_length = 45)
+    room_type_image = models.URLField(max_length = 2000)
```

Comment:
RoomType 이라고 하지만, 사실 Room 에 대한 정보만 담는 테이블인 것 같습니다.
```suggestion
class Room(models.Model):
    name            = models.CharField(max_length = 45)
    image_url = models.URLField(max_length = 2000)
```
이 더 깔끔해보이는데 어떻게 생각하시나요?

Reply:
수정 하겠습니다